pyproject.toml

```diff
@@ -78,6 +78,7 @@ types-python-dateutil = "^2.8.19.13"
 [tool.poetry.scripts]
 cz = "commitizen.cli:main"
 git-cz = "commitizen.cli:main"
+gen-cli-help-screenshot = "scripts.tasks:run_generate_help_screenshots"
```

Comment:
I think this script would make it to the final user, and instead we should simply use it from the scripts folders, what do you think?

Reply:
It's a good idea, thank you ~!!!

Reply:
We could make it a future improvement 👍